examples/proto/examplepb/a_bit_of_everything.proto

```diff
@@ -11,89 +11,106 @@ import "google/protobuf/timestamp.proto";
 import "protoc-gen-swagger/options/annotations.proto";
 
 option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
-  info: {
-    title: "A Bit of Everything";
-    version: "1.0";
-    contact: {
-      name: "gRPC-Gateway project";
-      url: "https://github.com/grpc-ecosystem/grpc-gateway";
-      email: "none@example.com";
-    };
-  };
-  // Overwriting host entry breaks tests, so this is not done here.
-  external_docs: {
-    url: "https://github.com/grpc-ecosystem/grpc-gateway";
-    description: "More about gRPC-Gateway";
-  }
-  schemes: HTTP;
-  schemes: HTTPS;
-  schemes: WSS;
-  consumes: "application/json";
-  consumes: "application/x-foo-mime";
-  produces: "application/json";
-  produces: "application/x-foo-mime";
-  security_definitions: {
-    security: {
-      key: "BasicAuth";
-      value: {
-        type: TYPE_BASIC;
-      }
-    }
-    security: {
-      key: "ApiKeyAuth";
-      value: {
-        type: TYPE_API_KEY;
-	in: IN_HEADER;
-	name: "X-API-Key";
-      }
-    }
-    security: {
-      key: "OAuth2";
-      value: {
-        type: TYPE_OAUTH2;
-	flow: FLOW_ACCESS_CODE;
-	authorization_url: "https://example.com/oauth/authorize";
-	token_url: "https://example.com/oauth/token";
-	scopes: {
-	  scope: {
-	    key: "read";
-	    value: "Grants read access";
-	  }
-	  scope: {
-	    key: "write";
-	    value: "Grants write access";
-	  }
-	  scope: {
-	    key: "admin";
-	    value: "Grants read and write access to administrative information";
-	  }
+	info: {
```

Comment:
Out of curiosity, why did you change everything to be tabs? Can you run this through a modern version of clang-format?

Reply:
Everything except for the examples added by @ivucica was already tabs, so I just made it consistent as it was driving me mad. I could format this (and the other protofiles) with `prototool format` [1] if you like, but I havne't touched the others.

[1]: https://github.com/uber/prototool#prototool-format

Reply:
(Sorry about this.)